apps/client/src/main.tsx

```diff
@@ -0,0 +1,16 @@
+import { StrictMode } from 'react';
+import * as ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+
+import App from './app/app';
```

Comment:
The Main.tsx file should be on the root folder of the client, that way you can easily identify the entry point of your application